source/data-api/examples.txt

```diff
@@ -0,0 +1,290 @@
+=====================
+{+data-api+} Examples
+=====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+The following examples demonstrate how to send requests to the Atlas
+{+data-api+}.
+
+Specify BSON Types in Requests
+------------------------------
+
+{+data-api+} requests can specify BSON types that don't exist in JSON by
+instead using the EJSON :ref:`data format <data-api-data-formats>` in
+the request body. You can use EJSON to match BSON types in query filters
+or write BSON types in insert and update operations.
+
+To specify that a request body uses EJSON, set the ``Content-Type`` header:
+
+.. code-block:: none
+   
+   Content-Type: application/ejson
```

Comment:
worth mentioning in here that uses plain ol JSON by default

Reply:
There actually is no default I believe - you must specify either JSON or EJSON (though both work identically atm)